python/pyarrow/tests/test_dataset.py

```diff
@@ -1569,14 +1570,6 @@ def test_partitioning_factory_segment_encoding():
         inferred_schema = factory.inspect()
 
 
-def test_dictionary_partitioning_outer_nulls_raises(tempdir):
-    table = pa.table({'a': ['x', 'y', None], 'b': ['x', 'y', 'z']})
-    part = ds.partitioning(
-        pa.schema([pa.field('a', pa.string()), pa.field('b', pa.string())]))
-    with pytest.raises(pa.ArrowInvalid):
-        ds.write_dataset(table, tempdir, format='parquet', partitioning=part)
```

Comment:
since there are other tests below about how nulls are treated in partition fields that is closely related to this one, I would leave this one test at this location

Reply:
put it back where it was